MTA-552 mta 6 2 0 release notes hvider (#713)
windup/windup-documentation: docs/topics/mta-rn-known-issues-6-2-0.adoc

// Module included in the following assemblies:
//
// * docs/release_notes/master.adoc

:_content-type: REFERENCE
[id="rn-known-issues-6-2-0_{context}"]
= Known issues

MTA version 6.2.0 has the following issues.

.MTA fails to run analysis in private GitHub repo
Fail to run analysis in private github repo. link:https://issues.redhat.com/browse/MTA-810[MTA-810]

.Jira connection (instance) cannot be removed 
Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  

> See an updated  filtered list of link:https://issues.redhat.com/issues/?filter=12418204[known issues in Jira].

Reviewer: @HagayVider1 - here is how I would like the Known Issues to look. What do you think?

Please reach out to Dylan and ask him if there are any other known issues he wants adding? Also ask for more details about the two known issues (if he wants them keeping)

Please can you check the Jira filter and see if you can figure out what is going wrong as I cannot access it.

// Module included in the following assemblies:
//
// * docs/release_notes/master.adoc

:_content-type: REFERENCE
[id="rn-known-issues-6-2-0_{context}"]
= Known issues

MTA version 6.2.0 has the following issues.

.MTA fails to run analysis in private GitHub repo

Fail to run analysis in private github repo. link:https://issues.redhat.com/browse/MTA-810[MTA-810]

.Jira connection (instance) cannot be removed 

> Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  

Reviewer: *Jira **connections...*** plural .... ***it was** added* is singular

> Jira connections cannot be removed after it was added to Wave even after Wave was removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  

Reviewer: ```suggestion
Jira connections cannot be removed after they are added to a _Migration Wave_, even after the Wave has been removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]  
```

// Module included in the following assemblies:
//
// * docs/release_notes/master.adoc

:_content-type: REFERENCE
[id="rn-known-issues-6-2-0_{context}"]
= Known issues

> MTA version 6.2.0 has the following issues.

Reviewer: Please can you follow the existing format for MTA - https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.1/html/release_notes/mta-6-1-0#rn-known-issues-610_release-notes

![image](https://github.com/windup/windup-documentation/assets/106804821/a451e9ec-ef38-4843-93b4-f518883d5983)

> * link:https://issues.redhat.com/browse/MTA-823[MTA-823] - Report custom metrics on MTA usage back via OpenShift Telemetry

Reviewer 2: @HagayVider1 
This not a known issue. This is a new feature.
Do you have a list of known features from QE?

Reviewer: Not yet! But we want to be ahead of the curve

> * link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Report custom metrics on MTA usage back via OpenShift Telemetry

Reviewer 2: @HagayVider1 
Same as above. This not a known issue. This is a new feature.

* link:https://issues.redhat.com/browse/MTA-402[MTA-402] - Downstream efforts for MTA 6.2.0
* link:https://issues.redhat.com/browse/MTA-894[MTA-894] - Failed analysis counted twice

> * link:https://issues.redhat.com/browse/MTA-872[MTA-872] - After an error, fails to export same applications as tastks, returns error showing sub-tasks

Reviewer: Use Vale or check for typos
```suggestion
* link:https://issues.redhat.com/browse/MTA-872[MTA-872] - After an error, fails to export same applications as **tastks**, returns error showing sub-tasks
```

* link:https://issues.redhat.com/browse/MTA-973[MTA-973] - Success alert missing when creating a new Jira instance
* link:https://issues.redhat.com/browse/MTA-810[MTA-810] - Fails to run analysis in provate GitHub repo

> * link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error retreiving data with `Unable to Connect`

Reviewer: * link:https://issues.redhat.com/browse/MTA-984[MTA-984] - Dependencies: error **retrieving** data with `Unable to Connect`

* link:https://issues.redhat.com/browse/MTA-696[MTA-696] - Exporting a migration wave to Jira causes all applications to enter Error state before creating issues

> * link:https://issues.redhat.com/browse/MTA-906[MTA-906] - Exporting Migration Wave as an Epic does not export to Jira - unsing Jira Server/DataCenter

Reviewer: **Unsing** or **Using**

* link:https://issues.redhat.com/browse/MTA-870[MTA-870] - A Migration Wave cannot be exported as a SubTask - using both Jira DataCenter and Cloud
* link:https://issues.redhat.com/browse/MTA-465[MTA-465] - Tags and Reports tabs for application keeps loading while analysis in progress

> * link:https://issues.redhat.com/browse/MTA-1014[MTA-1014] - Cannot remove Jira connection afater it was added to Wave even after Wave was removed

Reviewer: **afater** or **after**

* link:https://issues.redhat.com/browse/MTA-1015[MTA-1015] - Credential filtering is missing _Created by_ filter
* link:https://issues.redhat.com/browse/MTA-849[MTA-849] - Unable to connect to Jira instance behind a proxy

> See an updated  filtered list of link:https://issues.redhat.com/issues/?filter=12418204[known issues in Jira].

Reviewer 2: @HagayVider1 
This link doesn't seem to work.
Also, you don't have to give a list of the issues if you provide a link to the filter.
